src/index.html

```diff
@@ -1,17 +1,152 @@
+          autocomplete="off"
+        >
+        <br>
+        <label for="how_old">How old are You?</label>
+        <input
+          type="number"
+          id="how_old"
+          name="how_old"
+          min="1"
+          max="100"
+          value="12"
+        >
+        <br>
+        <label for="birth">Full date of birth:</label>
+        <input
+          type="date"
+          id="birth"
+          name="birth"
+        >
+        <br>
+        <label for="term">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="registration">
+        <legend>Registration</legend>
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          required
+        >
+        <br>
+        <label for="passwoprd">Password:</label>
+        <input
+          type="passwoprd"
+          id="passwoprd"
+          name="passwoprd"
+          minlength="5"
+          maxlength="10"
+          autocomplete="off"
+        >
+      </fieldset>
+      <fieldset class="fact">
+        <legend>An interesting fact about you!</legend>
+        <div class="add-cats-margin">
+          <span>Do you love cats?</span>
+          <input
+          id="love_cats_y"
+          type="radio"
+          name="love_cats"
+          value="y"
+          >
+          <label for="love_cats_y">Yes</label>
+          <input
+            id="love_cats_n"
+            type="radio"
+            name="love_cats"
+            value="n"
+          >
+          <label for="love_cats_n">No</label>
+        </div>
+        <label for="color">What is your favorite color?</label>
+        <input
+          type="color"
+          id="color"
+          name="color"
+        >
+        <br>
+        <label for="time">What time do you go to bed?</label>
+        <input
+          type="time"
+          id="time"
+          name="time"
+        >
+        <br>
+        <div class="input-block">
+          <label for="brand">What are your favorite brand of cars?</label>
+          <select name="brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <label for="out_work">How do you rate our work?</label>
+        <input
+          type="range"
+          id="out_work"
+          name="out_work"
+          value="0"
+        >
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div class="input-block">
+          <label for="comments">Comments:</label>
+          <textarea
+            name="comments"
+            id="comments"
+            placeholder="comments"
+            autocomplete="off">
+          </textarea>
```

Comment:
fix this behavior (look up the checklist)
<img width="288" alt="image" src="https://user-images.githubusercontent.com/102411130/231675586-24ec608b-6259-4da9-ada9-8e6a3fdc26ba.png">